gui/default/syncthing/core/syncthingController.js

```diff
@@ -521,6 +527,19 @@ angular.module('syncthing.core')
             console.log("recalcCompletion", device, $scope.completion[device]);
         }
 
+        function replaceAddressPort(address, newPort) {
+            var lastColonIndex = address.length;
+            for (var index = 0; index < address.length; index++) {
+                if (address[index] === ":") {
+                    var lastColonIndex = index;
+                }
+            }
+            if (newPort === "") {
```

Comment:
I assume this is no longer a string, so this condition is not needed?
I guess I lacked understanding what was the purpose of it before.

Reply:
done